src/Controls/src/Core/Platform/GestureManager/GesturePlatformManager.Windows.cs

```diff
@@ -30,7 +30,7 @@ class GestureManager : IDisposable
 		bool _wasPanGestureStartedSent;
 		bool _wasPinchGestureStartedSent;
 
-		public GestureManager(IViewHandler handler)
+		public GesturePlatformManager(IViewHandler handler)
```

Comment:
Should windows also use ToPlatform instead of PlatformView? like 45

Reply:
Not sure

The Windows Gestures were specifically designed to separate these two out. 

https://github.com/xamarin/Xamarin.Forms/blob/d998887f135622704254d4cfeaa000f5b915e64d/Xamarin.Forms.Platform.UAP/VisualElementRenderer.cs#L759-L766

So, the `Container` was the `VisualElementRenderer` and the `Control` was the inner control.

I'm not sure if that same requirement translates to our containers or not.  It looks like it was a way to implement `PreventGestureBubbling`. 

![image](https://user-images.githubusercontent.com/5375137/213337408-209bdb17-ca42-4325-8733-b3268020e2ce.png)

So, the `Container` would eat the event and stop it from passing through. It looks like this was implemented to try and make all the platforms act the same https://github.com/xamarin/Xamarin.Forms/pull/842. 

I'm guessing this no longer works like it did on Forms and I'm not really sure if we want it to. I'd say we create an issue to investigate, and I assume this will get addressed once issues related to InputTransparent or other scenarios come into focus

Reply:
https://github.com/dotnet/maui/issues/12828